### PR TITLE
added login and signup page

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added login and signup page
 - Added routes for both
 - Created general Input component and used that.

## Link:
https://deploy-preview-6--evolt-ecommerce.netlify.app/

## Screenshot:
image
image

### CODE REVIEW

**Review by SanketDhabarde — APPROVED** [review]
- Liked how u are handling form input change with only one handler.
- Good folder structure 
- liked using util.css for some common stylings

File: src/components/Input/Input.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	updateValue,
+	error,
+	showPassword,
+}) => {
+	return (
+		<div className="flex-column mb-3">
+			<label htmlFor={id}>{title}</label>
+			<input
+				required
+				id={id}
```
Comment (akshay-rajput) [inline src/components/Input/Input.jsx:19]: use default props or fallback values

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -54,14 +54,14 @@ export const Navbar = () => {
 
 				<ul className="navbar-options flex-row flex-center">
 					<li className="profile-icon relative">
-						<Link to="/login" className="flex-column items-center">
+						<div className="cursor-pointer">
```
Comment (akshay-rajput) [inline src/components/Navbar/Navbar.jsx]: revert this and conditionally render login or profile options

Comment (rohanmathur91) [reply src/components/Navbar/Navbar.jsx]: Done the changes.

File: src/components/Input/Input.jsx (hunk 1 above)
Comment (rohanmathur91) [reply src/components/Input/Input.jsx:19]: Done the changes.

**Review by ishitakeshawani — APPROVED** [review]
Why should use default props?

**Comment by rohanmathur91** [comment]
> Why should use default props?

Suppose we forgot to pass any prop to the component, so at that time we should not get an unexpected behavior or our app should not break so for that we have a default value.

**Pull request merged**